packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md

```diff
@@ -1,7 +1,17 @@
-To use the Okta CLI:
+From the Okta CLI:
 
 1. Enter the command: `okta apps create`.
-2. Give your app a name.
-3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application is using the tokens solely in server-side code.
-4. Select **Other**.
-5. For the **Login redirect URI**, enter: `http://localhost:8080/login/callback`.
+2. Type your app name.
+3. Select **Single Page App** or **Web** as the **Type of Application**.
+    - **Single Page App**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
+    - **Web**&mdash;for applications using tokens solely in server-side code
+
```

Comment:
There is a missing step here when creating a "Web" app using the cli:

```txt
Type of Application
> 1: Okta Spring Boot Starter
> 2: Spring Boot
> 3: JHipster
> 4: Quarkus
> 5: Other
Enter your choice [Other]: 
```

Using the default creates a `.okta.env` file in the current directory:

```sh
export OKTA_OAUTH2_ISSUER="https://<okta-url>/oauth2/default"
export OKTA_OAUTH2_CLIENT_ID="<client-id>"
export OKTA_OAUTH2_CLIENT_SECRET="<secret-is-here>"
```

The other "web" options will write the configuration directly to an application-specific location (for example Spring Boot is typical `src/main/resources/application.properties`

These values are NOT shown to the screen (they contain a secret), instead the user sees something like:

```txt
Okta application configuration has been written to: <current-directory-here>/.okta.env
```

SPA applications don't lend themselves well to this type of configuration (which is another reason why those values are just displayed on the screen)

Reply:
Thanks for the feedback! I've added these steps and notes to the CLI instructions.